_studio/shared/src/libmfx_core_vaapi.cpp

```diff
@@ -897,8 +921,10 @@ VAAPIVideoCORE::DoFastCopyWrapper(
 
     if (srcMemType & MFX_MEMTYPE_EXTERNAL_FRAME)
     {
+#ifndef MFX_VA_LINUX
```

Comment:
That's another PR: https://github.com/Intel-Media-SDK/MediaSDK/pull/1141. Don't mix.